strings/FrequencyOfCharacters

```diff
@@ -0,0 +1,23 @@
+import java.util.*;
+class Main{
+ public static void main(String[] args){
+   
+   frequencyOfCharacters("fngbgburfufufhurfjsfshsbpoowoimmnerjifidififif");
+ }
+ public static void frequencyOfCharacters(String str){
+    HashMap<Character,Integer>map=new HashMap<>();
+    for(int i=0;i<str.length();i++){
+       if(map.containsKey(str.charAt(i))){
+       map.put(str.charAt(i),map.get(str.charAt(i))+1);
+       }
+       else{
+       map.put(str.charAt(i),1);
+       }
+    }
+    Set<Character>key=map.keySet();
+    for(Character i:key){
+    System.out.println(i+" "+map.get(i));
```

Comment:
Instead of printing the values here, you could return the Map and let the caller handle the printing. That way this function becomes more modular and easily controllable.